### PR TITLE
[WIP] Update crashlytics sample for Crashlytics 4

### PR DESCRIPTION
**Pull request by morganchen12**
do not merge

### CODE REVIEW

File: crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -128,6 +128,13 @@
 			path = CrashlyticsExampleUITests;
 			sourceTree = "<group>";
 		};
+		3E68189DB1DD7B4868EE2558 /* Pods */ = {
```
Comment (paulb777) [inline crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj]: Don't check  the Pods stuff in.

File: crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -138,6 +145,7 @@
 				1073482720333B71004A66D1 /* CrashlyticsExampleUITests */,
 				5F5A534D1ADE670C00F81DF0 /* Products */,
 				5F9961041AE0CF4F0034F503 /* Shared */,
+				3E68189DB1DD7B4868EE2558 /* Pods */,
```
Comment (paulb777) [inline crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj]: And here.

Deleting the red Pods folders in Xcode is another way to manage this.

File: crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj (hunk 1 above)
Comment (morganchen12) [reply crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj]: Done.

File: crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj (hunk 2 above)
Comment (morganchen12) [reply crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj]: Done.

**Comment by morganchen12** [comment]
I was planning on just leaving the PR unmerged until 6.15.0 is released.

**Comment by morganchen12** [comment]
Merging since the crashlytics build passed (though Analytics failed).

**Pull request merged**